### ISSUE TITLE
Release TwistedNames-0.4

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#2361]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)| @radix reported|
|-|-|
|Trac ID|trac#2361|
|Type|enhancement|
|Created|2007-01-06 21:41:53Z|

<details><summary>Searchable metadata</summary>

```
trac-id__2361 2361
type__enhancement enhancement
reporter__radix radix
priority__normal normal
milestone__Names_0_4 Names-0.4
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__release_management release management
keywords__ 
time__1168119713000000 1168119713000000
changetime__1168135316000000 1168135316000000
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)<a name="note_1"></a>|@radix set owner to @exarkun|
|-|-|

Ready for review at releases/twisted-name-0.4.x-2361

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_2"></a>|@exarkun set owner to @radix|
|-|-|

Looks good.

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)<a name="note_3"></a>|@radix commented|
|-|-|

for reference I typod the branch name, it is: releases/twisted-names-0.4.x-2361

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)<a name="note_4"></a>|@radix set status to `closed`|
|-|-|

(In [19231]) Merge releases/twisted-names-0.4.x-2361

Author: release automation, radix
Reviewer: exarkun
Fixes [#2361](https://github.com/twisted/twisted/issues/2361)

Versions and NEWS from names 0.4.0

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
